bika/lims/content/analysisspec.py

```diff
@@ -226,6 +227,7 @@ def getSampleTypes(self):
 
         return DisplayList(sampletypes)
 
+    @deprecated('Flagged in 17.03')
```

Comment:
@Espurna pay attention to `deprecated`decorator. It works like a charm now!. I think that writting the year.month the function was labelled as deprecated (*Flagged in 17.03*) is enough